contrib/check-ifd-firmware.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2024 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# pylint: disable=invalid-name,missing-docstring,too-few-public-methods
+
+from typing import Dict
+import os
+import glob
+import sys
+import subprocess
+import json
+from collections import defaultdict
+
+
+class TermColor:
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
```

Comment:
I could see us using this in other scripts in the future.  What do you think about splitting it off into `fwupd_setup_helpers.py`?  Or maybe even renaming that to `fwupd_contrib_helpers.py`?

Reply:
I think `from termcolor import colored` is probably much better! Fixed.